xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/macros.vm

```diff
@@ -2322,6 +2328,28 @@ $string##
 #end
 
 
+#**
+* Pulls all the JavaScript and CSS resources needed by the page picker.
+*#
+#macro (pagePicker_import)
+  #picker_import
+  #set ($discard = $xwiki.jsfx.use('uicomponents/suggest/suggestPages.js',
+    {'forceSkinAction': true, 'language': $xcontext.locale}))
+#end
+
+#macro (pagePicker_input $parameters)
```

Comment:
Mostly to be more consistent with the `userPicker` and so that we can add other instructions in the macro but it's not very useful for now indeed. I'll remove it.